queries.sql

```diff
@@ -113,6 +113,69 @@ FROM species
 JOIN animals ON animals.species_id = species.id
 GROUP BY species.name;
 
+-- 1. Who was the last animal seen by William Tatcher?
+SELECT a.name AS animal_name
+FROM animals a
+JOIN visits v ON a.id = v.animal_id
+JOIN vets ve ON v.vet_id = ve.id
+WHERE ve.name = 'William Tatcher'
+ORDER BY v.date DESC
+LIMIT 1;
```

Comment:
- [ ] the last animal seen by William Tatcher should be blossom, however, your query returns an empty row.